src/vca_reviews/mod.rs

```diff
@@ -4,21 +4,14 @@ pub use tags::TagsMap;
 
 use vit_servicing_station_lib::db::models::community_advisors_reviews::AdvisorReview;
 
-use calamine::{open_workbook, Reader, Xlsx};
 use serde::Deserialize;
 
 use std::path::Path;
 
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
     #[error(transparent)]
-    CouldNotReadExcel(#[from] calamine::Error),
-
-    #[error(transparent)]
-    Xls(#[from] calamine::XlsxError),
-
-    #[error(transparent)]
-    CouldNotDeserialize(#[from] calamine::DeError),
+    CouldNotReadExcel(#[from] csv::Error),
```

Comment:
Well, it's not and excel file anymore :)

Reply:
indeed